hera_mc/sys_handling.py

```diff
@@ -314,3 +314,17 @@ def publish_summary(self, hlist=['HH'], rev='A', exact_match=False,
             return 'OK'
         else:
             return 'Not on "main"'
+
+    def system_comments(self, system_kw='System', kword='all', tabkw=7):
```

Comment:
I see what `tabkw` does here but the variable name is pretty opaque to me. Also, the one place that calls this function never passes a value for that keyword. It seems like it would be better to dynamically adjust the column size in the same way that the comment column is adjusted.